fusillade/clouddirectory.py

```diff
@@ -1735,25 +1735,23 @@ def get_users_page(self, next_token=None, per_page=None) -> Tuple[Dict, str]:
             next_token=next_token)
         return {'users': results}, next_token
 
-    def add_users(self, users: List['User']) -> None:
+    def add_users(self, users: List[str]) -> None:
         if users:
-            User.exists([user.name for user in users])
-            operations = [i for i in itertools.chain(*[user.add_groups([self.name], False) for user in users])]
+            operations = [i for i in itertools.chain(*[User(user).add_groups([self.name], False) for user in users])]
             self.cd.batch_write(operations)
             logger.info(dict(message="Adding users to group",
                              object=dict(type=self.object_type, path_name=self._path_name),
-                             users=[user._path_name for user in users]))
+                             users=[user for user in users]))
```

Comment:
Given `users: List[str]`, is there a reason that this isn't just

```suggestion
                             users=users))
```
?

Reply:
This construct looks like it would be useful if `users` were produced by a generator, in which case `list(users)` could be used to extract all values from the generator and turn them into a list

Reply:
This line was previously `users=[user._path_name for user in users]` and `_path_name` was removed, so it can be changed to just `users=users`